### ISSUE TITLE
Cannot install this plugin on latest curity

### ISSUE THREAD

**Issue by nekolab** [curityio/kong-dev-portal-user-provisioner#3]
Hi, I've received these error messages while trying install kong dev portal plugin in Curity

```
curity  | openjdk version "17.0.6.0.101" 2023-04-18 LTS
curity  | OpenJDK Runtime Environment Zulu17.41+14-SA (build 17.0.6.0.101+2-LTS)
curity  | OpenJDK 64-Bit Server VM Zulu17.41+14-SA (build 17.0.6.0.101+2-LTS, mixed mode)
curity  | 2023-08-11T02:54:39:239+0000 INFO    {main} se.curity.identityserver.app.LogInitializer - Using log4j config: XmlConfiguration[location=/opt/idsvr/etc/log4j2.xml]
curity  | 2023-08-11T02:54:39:244+0000 INFO    {main} se.curity.identityserver.app.Bootstrapper - Using default key for configuration encryption
curity  | 2023-08-11T02:54:39:566+0000 INFO    {server-start-stop-1} se.curity.identityserver.jetty.AppServer.node-status-server - Starting HTTP server (run 1)
curity  | 2023-08-11T02:54:39:585+0000 INFO    {main} se.curity.identityserver.app.Bootstrapper - Identity Server version 8.3.2
curity  | 2023-08-11T02:54:39:646+0000 WARN    {main} se.curity.identityserver.app.Bootstrapper - Caught an unhandled exception loading plug-ins
curity  | se.curity.identityserver.prebooter.PluginBuildException: Failed to update plugin group 'kong'
curity  | 	at se.curity.identityserver.prebooter.PluginGroupManager.compileAndUpdatePluginGroup(PluginGroupManager.kt:177) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.PluginGroupManager.compileAndUpdateAllPluginGroups(PluginGroupManager.kt:114) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.PluginInitializer$initializePlugins$compiledPlugins$1.invoke(PluginInitializer.kt:39) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.PluginInitializer$initializePlugins$compiledPlugins$1.invoke(PluginInitializer.kt:38) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at kotlin.sequences.FlatteningSequence$iterator$1.ensureItemIterator(Sequences.kt:315) ~[kotlin-stdlib-1.6.10.jar:1.6.10-release-923(1.6.10)]
curity  | 	at kotlin.sequences.FlatteningSequence$iterator$1.hasNext(Sequences.kt:303) ~[kotlin-stdlib-1.6.10.jar:1.6.10-release-923(1.6.10)]
curity  | 	at kotlin.sequences.SequencesKt___SequencesKt.toCollection(_Sequences.kt:786) ~[kotlin-stdlib-1.6.10.jar:1.6.10-release-923(1.6.10)]
curity  | 	at kotlin.sequences.SequencesKt___SequencesKt.toMutableList(_Sequences.kt:816) ~[kotlin-stdlib-1.6.10.jar:1.6.10-release-923(1.6.10)]
curity  | 	at kotlin.sequences.SequencesKt___SequencesKt.toList(_Sequences.kt:807) ~[kotlin-stdlib-1.6.10.jar:1.6.10-release-923(1.6.10)]
curity  | 	at se.curity.identityserver.prebooter.PluginInitializer.initializePlugins(PluginInitializer.kt:40) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.PreBooter.runPrebooter(PreBooter.kt:36) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.app.Bootstrapper.main(Bootstrapper.java:255) ~[identityserver.host-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.app.Bootstrapper.main(Bootstrapper.java:107) ~[identityserver.host-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | Caused by: java.lang.IllegalArgumentException: The /opt/idsvr/usr/share/plugins/kong/slf4j-api-1.7.36.jar file provides the slf4j.api library with version 1.7.36, but the server only supports version 2.0.3. Please change the version of this library the plugin is compiled with.
curity  | 	at se.curity.identityserver.prebooter.loader.PluginClassLoaderFactoryKt.validateAndFilterJars(PluginClassLoaderFactory.kt:124) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.loader.PluginClassLoader.<init>(PluginClassLoader.kt:55) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.loader.PluginClassLoaderFactory.apply(PluginClassLoaderFactory.kt:77) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.PluginGroupManager.compilePluginsInGroup(PluginGroupManager.kt:184) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	at se.curity.identityserver.prebooter.PluginGroupManager.compileAndUpdatePluginGroup(PluginGroupManager.kt:164) ~[identityserver.yang-generator-8.3.2-70cbd834f8.jar:8.3.2-70cbd834f8]
curity  | 	... 12 more
curity  | 2023-08-11T02:54:39:646+0000 INFO    {main} se.curity.identityserver.app.DelegatableShutdownSwitch - Server being shutdown by switch called 'plugin-loader'
```
I noticed, according to the doc, this plugin only supports Curity 6.0 and below, does it mean the latest Curity is not supported?

Thanks

**Comment by mtrojanowski** [comment]
Hi @nekolab, thanks for pointing this out!

Turns out the `mvn package` command was needlessly copying the slf4j jar to the target directory. When you tried to deploy the server the dependencies clashed as there is now a newer version of slf4j provided by the server itself. I opened a PR with a fix (#1) but you can also just remove the slf4j jar from the plugin directory and the server will start.

**Comment by mtrojanowski** [comment]
I merged the PR, so closing the issue. If you encounter any more problems, please feel free to comment here or open a new issue.
